doc/cephadm/install.rst

```diff
@@ -45,7 +46,7 @@ curl-based installation
   
   .. prompt:: bash #
 
-   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/|stable-release|/src/cephadm/cephadm
```

Comment:
need to add `:substitutions:` after `.. prompt:: bash #`

see https://github.com/ceph/ceph/blob/master/doc/install/get-packages.rst#debian-packages .

Reply:
like

```rst
.. prompt:: bash $
   :substitutions:

   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/|stable-release|/src/cephadm/cephadm
```